packages/runtime-common/realm.ts

```diff
@@ -387,7 +379,7 @@ export class Realm {
         this.removeCard.bind(this),
       )
       .post(
-        `/.+(${this.writableFileExtensions.map((e) => '\\' + e).join('|')})`,
+        '/.*',
```

Comment:
I lacked the confidence for such a bold move when looking at this from the extension side 🙃